pkg/virt-api/webhooks/validating-webhook/admitters/pod-eviction-admitter_new_test.go

```diff
@@ -104,6 +110,149 @@ var _ = Describe("Pod eviction admitter", func() {
 			Expect(kubeClient.Fake.Actions()).To(HaveLen(1))
 		})
 	})
+
+	DescribeTable("Handle virt-launcher eviction", func(clusterWideEvictionStrategy, vmiEvictionStrategy *kvirtv1.EvictionStrategy, isVMIMigratable, expectVMIEvacuation bool) {
```

Comment:
I find the following more readable:
```
func(clusterWideEvictionStrategy, expectVMIEvacuation bool, vmiOptions ...vmiOption)
```

then the `Entry` is also self-descriptive 
```
Entry("When cluster-wide eviction strategy is nil, VMI eviction strategy is LiveMigrate and VMI is migratable - Trigger VMI Evacuation",
			nil,
			true,
			withEvictionStrategy(pointer.P(kvirtv1.EvictionStrategyLiveMigrate)),
			withLiveMigratableCondition()
		),
```

Last but not least, we can also pass a `matcher` instead of `expectVMIEvacuation` if we follow Federico's work.

Reply:
Changed the arguments to `vmiOption` as requested.
Dropped the `expectVMIEvacuation` boolean in favor of two tables.

This boolean is the same as in:
```go
func f(b bool) {
  if b {
    doThis()
  } else {
    doThat()
  }
}
```